client/aggregator_test.go

```diff
@@ -231,6 +231,7 @@ func TestClient_UpdateAggregator(t *testing.T) {
 
 	err = withToken.UpdateAggregator(ctx, created.ID, types.UpdateAggregator{
 		Name: ptrStr("test-aggregator-updated"),
+		Tags: ptrSliceStr([]string{"updatedtag"}),
```

Comment:
Please add some extra test with empty, nil, " ", "\t " and multiple tags.

Reply:
Is it really needed that much validation here?

Actual testing happens on the server.
These are the client tests, we just need to know that the endpoint is connected.

I updated the test to refetch the core instance after the update and check that it was indeed updated.

Reply:
I'm totally ok moving the tests here btw 
@niedbalski 
It would be a lot less of code repetition.
But the flow of doing two PRs on different repos prevent it :(